### PR TITLE
Migrate customer view actions

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR migrates  "Save private note" & "Transform guest to customer" actions.
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/11010
| How to test?  | Access `admin-dev/index.php/sell/customers/1/view` and check migrated actions that should work the same as in legacy.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11527)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig [hunk 1]
```diff
@@ -169,7 +169,7 @@
           {% if customerInformation.generalInformation.customerBySameEmailExists %}
             <p>{{ 'A registered customer account using the defined email address already exists. '|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig:170]: Watch the space in the end. ;-)

File: src/Core/Domain/Customer/ValueObject/CustomerId.php [hunk 2]
```diff
@@ -66,6 +66,6 @@ private function setCustomerId($customerId)
             ));
         }
 
-        $this->customerId = $customerId;
+        $this->customerId = (int) $customerId;
```
Comment (mickaelandrieu) [inline src/Core/Domain/Customer/ValueObject/CustomerId.php]: you shouldn't have to do that, improve the conditions of input validity instead.

File: src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php [hunk 3]
```diff
@@ -142,4 +148,113 @@ public function viewAction($customerId, Request $request)
...
+     * )
+     *
+     * @param int $customerId
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function savePrivateNoteAction($customerId, Request $request)
+    {
+        $privateNoteForm = $this->getPrivateNoteForm();
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php]: useless function create your form directly.

**Review by mickaelandrieu — DISMISSED** [review]
minor changes requested => good job!

**Comment by matks** [comment]
After mickaelandrieu I have nothing else to add 😄

**Comment by sarjon** [comment]
@matks all done. :+1:

**Review by matks — DISMISSED** [review]

**Comment by matks** [comment]
QA it is ! 👏

**Comment by marionf** [comment]
@TristanLDD is it ok for you ?

capture d ecran_691

**Comment by TristanLDD** [comment]
Hello ! There is a little modification to do on the text above the blue button: text size should be reduced to 13px

capture d ecran 2018-12-10 a 10 39 41

Otherwise, everything is ok for me !

**Comment by sarjon** [comment]
@TristanLDD is this size text part of UI Kit? i can find `small-text` which is 12px or `xsmall-text` which is 10px. Is there a specific name for this size of text? or should i create new text style (CSS) specific to this page only?

**Comment by TristanLDD** [comment]
@sarjon My bad, you can use small-text ;)

**Comment by sarjon** [comment]
and its done. :) you can take a look. https://prnt.sc/lt559u

**Comment by TristanLDD** [comment]
@sarjon Thank you !

**Review by matks — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @sarjon and everyone!

**Pull request merged**